code/web/src/modules/user/Profile.js

```diff
@@ -18,82 +18,120 @@ import { logout } from './api/actions'
-          style={{ 
-            marginBottom: '2em', 
-            height: '24em', 
-            boxShadow: level1, 
-            borderRadius: '10px'
+const Profile = ({ user, logout }) => {
+  const { details } = user
+
+  return (
+    <div>
+      {/* SEO */}
+      <Helmet>
+        <title>My Profile - Crate</title>
+      </Helmet>
+
+      {/* Top title bar */}
+      <Grid style={{ backgroundColor: grey }} justifyRight={true}>
+        <GridCell
+          style={{
+            padding: "2em",
+            textAlign: "center",
+            maxWidth: "20vw",
+            marginRight: "24%",
           }}
-        /> 
-        <H4>{props.user.details.name}</H4>
-      </GridCell>
+        >
+          <H3 font="secondary">My profile</H3>
+        </GridCell>
 
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
-        <H4 font='secondary' style={{ marginBottom: '1.5em' }}>Account Details</H4>
+        <GridCell
+          style={{ padding: "2em", textAlign: "center", maxWidth: "9vw" }}
+        >
+          <Button
+            theme="secondary"
+            onClick={logout}
+            style={{ marginLeft: "1em" }}
+          >
+            Logout
+          </Button>
+        </GridCell>
+      </Grid>
 
-        <H6 style={{ marginBottom: '0.5em' }}>Account email:</H6>
-        <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.email}</p>
+      <Grid>
+        <GridCell style={{ padding: "2em", textAlign: "center" }}>
+          <img
+            src={details.image || `${APP_URL}//images/stock/men/4.jpg`}
+            style={{
+              marginBottom: "2em",
+              height: "24em",
+              boxShadow: level1,
+              borderRadius: "10px",
+            }}
+          />
+          <H4>{details.name}</H4>
+        </GridCell>
 
-        <H6 style={{ marginBottom: '0.5em' }}>Shipping Address:</H6>
-        <p style={{ color: grey2, marginBottom: '2em' }}>
-          123 Vanilla St.<br/>Chocolate, CO<br/>80911
-        </p>
+        <GridCell style={{ padding: "2em", textAlign: "center" }}>
+          <H4 font="secondary" style={{ marginBottom: "1.5em" }}>
+            Account Details
+          </H4>
 
-        <H6 style={{ marginBottom: '0.5em' }}>Bio:</H6>
-        <p style={{ color: grey2, width: '60%', margin: '0 auto 2em' }}>
-          "Clothes are just accessories for my rollerblades, and I'm all about accessories."
-        </p>
+          <H6 style={{ marginBottom: "0.5em" }}>Account email:</H6>
+          <p style={{ color: grey2, marginBottom: "2em" }}>
+            {details.email}
+          </p>
 
-        <Link to={userRoutes.editProfile.path}>
-          <Button theme="primary">Edit Profile Info</Button>
-        </Link>
-      </GridCell>
+          <H6 style={{ marginBottom: "0.5em" }}>Shipping Address:</H6>
+          <p style={{ color: grey2, marginBottom: "2em" }}>
+            {details.street && 
+              <>
+                {details.street}
```

Comment:
I'm glad we split these up for you here on the FE. This looks like its smooth to access.